mobile/params.go

```diff
@@ -71,14 +73,25 @@ func TestnetGenesis() string {
 
 // ChainConfig is the core config which determines the blockchain settings.
 type ChainConfig struct {
-	ChainID        int64 // Chain ID for replay protection
-	HomesteadBlock int64 // Homestead switch block
-	DAOForkBlock   int64 // TheDAO hard-fork switch block
-	DAOForkSupport bool  // Whether the nodes supports or opposes the DAO hard-fork
-	EIP150Block    int64 // Homestead gas reprice switch block
-	EIP150Hash     Hash  // Homestead gas reprice switch block hash
-	EIP155Block    int64 // Replay protection switch block
-	EIP158Block    int64 // Empty account pruning switch block
+	ChainID        int64      // Chain ID for replay protection
+	HomesteadBlock int64      // Homestead switch block
+	DAOForkBlock   int64      // TheDAO hard-fork switch block
+	DAOForkSupport bool       // Whether the nodes supports or opposes the DAO hard-fork
+	EIP150Block    int64      // Homestead gas reprice switch block
+	EIP150Hash     Hash       // Homestead gas reprice switch block hash
+	EIP155Block    int64      // Replay protection switch block
+	EIP158Block    int64      // Empty account pruning switch block
+	PoWConfig      *PoWConfig // Consensus rules for the proof-of-work engine
+	PoAConfig      *PoAConfig // Consensus rules for the proof-of-authority engine
+}
+
+// PoWConfig is the consensus engine configs for proof-of-work based sealing.
+type PoWConfig struct{}
```

Comment:
It's not in this pr. The poa will need a variation of this. I need to move it out from this commit.